modules/rostests/unittests/interop/LocaleTests.cpp

```diff
@@ -340,7 +334,11 @@ static void TEST_LocaleTests(void)
         mod[m] = LoadLibraryExW(lib[m], NULL, LOAD_LIBRARY_AS_DATAFILE);
         if (!mod[m])
         {
-            trace("Failed to load '%S', error %lu\n", lib[m], GetLastError());
+            // Special case explorer_old, which exists on ReactOS only.
+            if (m == explorer_old && !IsReactOS())
+                skip("Failed to load '%S', error %lu. Will skip related checks\n", lib[m], GetLastError());
+            else
+                ok(FALSE, "Failed to load '%S', error %lu. Will skip related checks\n", lib[m], GetLastError());
```

Comment:
It feels like `explorer_old` should be made an optional testcase (i.e. skip it silently if it's not found), because we may drop it one day.

Reply:
I think the former should be postponed until the latter actually happens: keep the unittest/situation consistent.
Even if RosTests can be built without RosApps, I think a rare "false" report is better than a usual "missed" one.